assets/app/features/upgrade/controllers/upgrade-landing.controller.spec.js

```diff
@@ -91,6 +111,29 @@ describe('Upgrade Landing Controller', function() {
                     assert.isFalse(value.status);
                 });
             });
+
+            it('should contain checks for addons', function () {
+                expect(controller.prechecks.checks).toEqual(
+                    jasmine.objectContaining({
+                        'clusters_healthy': jasmine.anything(),
```

Comment:
I checked again and contrary to what I found before, `jasmine.any(Boolean)` works just fine as well.